spaces/domains.json

```
@@ -1,4 +1,5 @@
 {
+  "governance.cyberestate.io": "Cyberrealestate.eth"
```

ENS must be in lower case. Uppercase is not allowed.
```suggestion
  "governance.cyberestate.io": "cyberrealestate.eth"
```

---

fixed Thank you for your help I really appreciate it.